src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/controllers/SasTokenControllerTest.java

```diff
@@ -0,0 +1,71 @@
+package uk.gov.hmcts.reform.blobrouter.controllers;
+
+import com.azure.storage.common.Utility;
+import com.azure.storage.common.implementation.StorageImplUtils;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit.jupiter.web.SpringJUnitWebConfig;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import uk.gov.hmcts.reform.blobrouter.exceptions.ServiceConfigNotFoundException;
+
+import java.time.OffsetDateTime;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringJUnitWebConfig
+@SpringBootTest
+@AutoConfigureMockMvc
```

Comment:
same here

Reply:
Can't apply the same suggestion here. As this test is failing to initiate service class with `WebMvcTest` annotation